storage/aptosdb/src/pruner/ledger_store/ledger_store_pruner.rs

```diff
@@ -116,72 +122,94 @@ impl DBPruner for LedgerPruner {
             .set(target_version as i64);
     }
 
-    fn target_version(&self) -> Version {
-        self.target_version.load(Ordering::Relaxed)
+    fn record_progress(&self, min_readable_version: Version) {
+        self.record_progress_impl(
+            min_readable_version,
+            /*has_pending_work_before_min_readable_version=*/ false,
+        );
     }
 
-    fn record_progress(&self, min_readable_version: Version) {
-        self.min_readable_version
-            .store(min_readable_version, Ordering::Relaxed);
-        PRUNER_VERSIONS
-            .with_label_values(&["ledger_pruner", "min_readable"])
-            .set(min_readable_version as i64);
+    fn target_version(&self) -> Version {
```

Comment:
nit: move  this back up before `record_progress()`? (why change the order?)

Reply:
done